scripts/update-size.test.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const root = path.join(__dirname, '..');
+const pkg = require(path.join(root, 'package.json'));
+const file = path.join(root, pkg.browser[pkg.module]);
+const size = require('./update-size');
+
+describe('size-limit', () => {
+  it('check size to be object', () => {
+    return size
+      .sizeLimit(file, { ignore: ['react', 'prop-types'] })
+      .then(result => {
+        expect(result).toBeInstanceOf(Object);
+      });
+  });
+
+  it('check gzip size limit', () => {
+    return size
+      .sizeLimit(file, { ignore: ['react', 'prop-types'] })
+      .then(result => {
+        expect(result.gzip).toBe(1673);
```

Comment:
this also is not something that we want to test, this number is already written to the README and this test would fail with any change made to the source,  the only thing that we need to test is the threshold size specified in `.size-limit.js` and thats what is going to be tested with `size-limit` tool